CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # ChangeLog
 
+## Unreleased
+
+### New Features
+- New `TextNode`/`Document` object classes based on pydantic (#6586)
+- `TextNode`/`Document` objects support metadata customization (metadata templates, exclude metadata from LLM or embeddings) (#6586)
+- Nodes no longer require flat metadata dictionaries, unless the vector store you use requires it (#6586)
+
+### Breaking/Deprecated API Changes
+- `Node` has been renamed to `TextNode` and is imported from `llama_index.schema` (#6586)
+- `TextNode` and `Document` must be instansiated with kwargs: `Document(text=text)` (#6586)
+- `TextNode` (fka `Node`) has a `id_` or `node_id` property, rather than `doc_id` (#6586)
```

Comment:
very nit: typo on `aka`?

Reply:
`fka` = formally known as (there is technically no object called `Node` now)